### PR TITLE
Fix #7724: DataTable: components can't overflow in editable cells

### LINKED ISSUE #7724: DataTable: components can't overflow in editable cells
See https://github.com/primefaces-extensions/primefaces-extensions/issues/511

Screenshot 2021-08-20 at 11 58 36

This CSS rule sets overflow to hidden:

https://github.com/primefaces/primefaces/blob/65de61e60e6218b28ff147f947ed422e1ffd8916/primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.css#L29-L37

Shouldn't it be visible at least when a cell is editable?

https://github.com/primefaces/primefaces/blob/65de61e60e6218b28ff147f947ed422e1ffd8916/primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.css#L195-L201

### PR DESCRIPTION
**Pull request by jepsar**
Fix #7724

### CODE REVIEW

File: primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.css [hunk 1]
```diff
@@ -1,13 +1,9 @@
 .ui-datatable table {
-	border-collapse:collapse;
+    border-collapse: collapse;
     width: 100%;
     table-layout: fixed;
 }
 
-.ui-datatable-tablewrapper {
```
Comment (jepsar) [inline primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.css:7]: This was needed to allow fields close to the top or bottom of the table to overflow

**Comment by tandraschko** [comment]
We should Not apply such changes without Mert
It may breake cases again

**Comment by melloware** [comment]
I assigned to Mert.

**Comment by jepsar** [comment]
> We should Not apply such changes without Mert It may breake cases again

Yes, especially the change on the wrapper. I tested with the test repo and the showcase and there it is looking good. Happy to get feedback from Mert.

**Comment by mertsincan** [comment]
My comment: https://github.com/primefaces/primefaces/issues/7724#issuecomment-957368921

**Pull request closed without merging**